triggers/incident-triggered/event.schema.json

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "The incident's unique identifier"
+    },
+    "incidentNumber": {
+      "type": "integer",
+      "description": "The account-unique incident number"
+    },
+    "title": {
+      "type": "string",
+      "description": "A succinct description of the incident"
+    },
+    "urgency": {
+      "type": "string",
+      "description": "The incident's urgency",
+      "enum": ["high", "low"]
```

Comment:
We'll have to make sure @sprokusk can render this.

Reply:
It's OK if it doesn't get rendered for now - it's additive in terms of documentation information anyway.